### PR TITLE
feat: Add objectType and size to Unit

### PR DESCRIPTION
**Pull request by TomerRon**
This PR adds:
- `EnumUnitObjectType` enum
- `objectType` field to Unit
- `size` field to Unit

### CODE REVIEW

File: src/rest/methods/unit.ts [hunk 1]
```diff
@@ -1,5 +1,78 @@
 import { EntityResultList, IAllthingsRestClient } from '../types'
 
+export enum EnumUnitObjectType {
```
Comment (TomerRon) [inline src/rest/methods/unit.ts]: Looking for opinions about this enum, it's huge and really bloats this file. I was considering creating a new file for this enum, but didn't want to diverge from our current pattern.

File: src/rest/methods/unit.ts [hunk 2]
```diff
@@ -9,12 +82,14 @@ export interface IUnit {
   readonly externalId: string | null
   readonly id: string
   readonly name: string
+  readonly objectType: EnumUnitObjectType
+  readonly readOnly: boolean
+  readonly size: number | null
```
Comment (TomerRon) [inline src/rest/methods/unit.ts]: Looking for feedback about these fields - as far as I know, `size` is optional and `objectType` is required.

Comment (janrembold) [reply src/rest/methods/unit.ts]: Both types are not required by the API: https://github.com/allthings/php-prod/blob/master/symfony/src/Qipp/HomeBundle/Form/Type/UnitFormType.php#L31
and https://github.com/allthings/php-prod/blob/master/symfony/src/Qipp/HomeBundle/Form/Type/UnitFormType.php#L44

Comment (janrembold) [reply src/rest/methods/unit.ts]: Just talked to @peinwag and found out that only `name` and `type` are required fields in `Unit`

**Pull request merged**